test/plugin/base_test.rb

```diff
@@ -302,50 +302,150 @@ def test_structured_payload_malformatted_log
     end
   end
 
-  def test_structured_payload_json_log
+  def test_structured_payload_json_log_default_not_parsed_text
     setup_gce_metadata_stubs
+    json_string = '{"msg": "test log entry 0", "tag2": "test", ' \
+                  '"data": 5000, "some_null_field": null}'
     setup_logging_stubs do
-      d = create_driver
-      json_string = '{"msg": "test log entry 0", "tag2": "test", "data": 5000}'
+      d = create_driver(APPLICATION_DEFAULT_CONFIG)
       d.emit('message' => 'notJSON ' + json_string)
       d.emit('message' => json_string)
-      d.emit('message' => "\t" + json_string)
-      d.emit('message' => '  ' + json_string)
+      d.emit('message' => "  \r\n \t" + json_string)
+      d.run
+    end
+    verify_log_entries(3, COMPUTE_PARAMS, 'textPayload') do
+      # Only check for the existence of textPayload.
+    end
+  end
+
+  def test_structured_payload_json_log_default_not_parsed_json
+    setup_gce_metadata_stubs
+    json_string = '{"msg": "test log entry 0", "tag2": "test", ' \
+                  '"data": 5000, "some_null_field": null}'
+    setup_logging_stubs do
+      d = create_driver(APPLICATION_DEFAULT_CONFIG)
+      %w(log msg).each do |field|
```

Comment:
`%w(message log msg)`?

Reply:
No, `message` is special, because it ends up in `textPayload`, so it has a separate test above (`test_structured_payload_json_log_default_not_parsed_text`).

Reply:
Cool cool. Thanks for clarifiying.